cg/cli/workflow/microsalt/base.py

```diff
@@ -222,48 +184,23 @@ def start(
     context.invoke(run, ticket=ticket, sample=sample, unique_id=unique_id, dry_run=dry_run)
 
 
-@microsalt.command("store-available")
-@OPTION_DRY_RUN
-@click.pass_context
-def store_available(context: click.Context, dry_run: bool) -> None:
-    """Store all finished analyses in Housekeeper"""
-    microsalt_analysis_api: MicrosaltAnalysisAPI = context.obj["microsalt_analysis_api"]
-    exit_code: int = EXIT_SUCCESS
-
-    for case_obj in microsalt_analysis_api.get_deliverables_to_store():
-        LOG.info("Storing deliverables for %s", case_obj.internal_id)
-        if dry_run:
-            continue
-        try:
-            context.invoke(store, unique_id=case_obj.internal_id)
-        except Exception:
-            exit_code = EXIT_FAIL
-
-    if exit_code:
-        raise click.Abort
-
-
 @microsalt.command("start-available")
 @OPTION_DRY_RUN
 @click.pass_context
-def start_available(context: click.Context, dry_run: bool) -> None:
-    """Start whole microSALT workflow for all newly sequenced cases"""
-    microsalt_analysis_api: MicrosaltAnalysisAPI = context.obj["microsalt_analysis_api"]
+def start_available(context: click.Context, dry_run: bool = False):
```

Comment:
All these are so similar, is there any way to merge the code into one function?

Reply:
Cant do this unfortunately. This invokes a method in its local scope...